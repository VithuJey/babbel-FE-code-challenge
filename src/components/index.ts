import Layout from "./layout/Layout";
import Card from "./card/Card";
import Modal from "./modal/Modal";

export { Layout, Card, Modal };
