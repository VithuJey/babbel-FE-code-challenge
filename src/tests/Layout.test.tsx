import React from "react";
import { create } from "react-test-renderer";
import { Layout } from "../components";

describe("Layout component", () => {
  test("Matches the snapshot", () => {
    const layout = create(
      <Layout>
        <div></div>
      </Layout>
    );
    expect(layout.toJSON()).toMatchSnapshot();
  });
});
