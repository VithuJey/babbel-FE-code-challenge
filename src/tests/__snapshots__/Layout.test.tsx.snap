// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component Matches the snapshot 1`] = `
<div
  className="container"
>
  <div>
    <div />
  </div>
</div>
`;
